recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
```

Comment:
There's still no ICU 56 on Linux, and updating it requires recompiling Boost. Unfortunately I don't have time to do that.

What should we do?